pkg/skuba/actions/cluster/status/status.go

```diff
@@ -34,7 +35,25 @@ func Status(client clientset.Interface) error {
 		return errors.Wrap(err, "could not retrieve node list")
 	}
 
-	outputFormat := "custom-columns=NAME:.metadata.name,OS-IMAGE:.status.nodeInfo.osImage,KERNEL-VERSION:.status.nodeInfo.kernelVersion,KUBELET-VERSION:.status.nodeInfo.kubeletVersion,CONTAINER-RUNTIME:.status.nodeInfo.containerRuntimeVersion,HAS-UPDATES:.metadata.annotations.caasp\\.suse\\.com/has-updates,HAS-DISRUPTIVE-UPDATES:.metadata.annotations.caasp\\.suse\\.com/has-disruptive-updates,CAASP-RELEASE-VERSION:.metadata.annotations.caasp\\.suse\\.com/caasp-release-version"
+	for _, node := range nodeList.Items {
+		for label, value := range node.Labels {
+			_ = value
+			if strings.Contains(label, "node-role.kubernetes.io") && len(strings.Split(label, "/")) > 0 {
+				node.Labels["caasp-role.kubernetes.io"] = strings.Split(label, "/")[1]
```

Comment:
:thinking: hmmmmmmmm will this give the same output as kubectl? aka, `<none>` for "worker" nodes or do we need an `else` statement?

Reply:
never mind. i'm a dumb dumb.

Reply:
if you set  custom label, "kubectl label nodes worker-1 node-role.kubernetes.io/worker=", you will get 
```
skuba cluster status
** This is an UNTAGGED version and NOT intended for production usage. **
NAME       ROLE     OS-IMAGE                              KERNEL-VERSION           KUBELET-VERSION   CONTAINER-RUNTIME   HAS-UPDATES   HAS-DISRUPTIVE-UPDATES   CAASP-RELEASE-VERSION
master-0   master   SUSE Linux Enterprise Server 15 SP1   4.12.14-197.29-default   v1.16.2           cri-o://1.16.1      yes           no                       <none>
worker-0   worker   SUSE Linux Enterprise Server 15 SP1   4.12.14-197.29-default   v1.16.2           cri-o://1.16.1      yes           no                       <none>
worker-1   <none>   SUSE Linux Enterprise Server 15 SP1   4.12.14-197.29-default   v1.16.2           cri-o://1.16.1      yes           no                       <none>
worker-2   <none>   SUSE Linux Enterprise Server 15 SP1   4.12.14-197.29-default   v1.16.2           cri-o://1.16.1      yes           no                       <none>

kubectl get nodes
NAME       STATUS                     ROLES    AGE     VERSION
master-0   Ready                      master   4d20h   v1.16.2
worker-0   Ready                      worker   4d20h   v1.16.2
worker-1   Ready                      <none>   4d20h   v1.16.2
worker-2   Ready                      <none>   2d16h   v1.16.2
```

Reply:
>  hmmmmmmmm will this give the same output as kubectl? aka, `<none>` for "worker" nodes or do we need an `else` statement?

If there is no label of caasp-role.kubernetes.io, the role will be shown as None. However, what ever roles are assigned, it will be shown like `kubectl get node`.